components/BarcodeScanner.vue

```diff
@@ -66,10 +69,13 @@ export default {
       cameraList.value = availableCameras.map(c => {
         return {deviceId: c.id, label: c.label};
       });
-
-      // select last camera in list to attempt to default to a back-facing,
-      // non-fish-eye camera
-      const defaultCameraId = cameraList.value.at(-1).deviceId;
+      // Attempt to default to a back-facing, non-fish-eye camera
+      // Most often this is the last one in the list of back cameras
+      const backCameras = cameraList.value.filter(camera => {
+        return camera.label.toLowerCase().includes('back');
+      });
```

Comment:
Handle case that `label` isn't set and shorten to remove `return`.

```suggestion
      const backCameras = cameraList.value
        .filter(camera => camera.label?.toLowerCase().includes('back'));
```